Dockerfile

```diff
@@ -16,26 +16,39 @@
 
 FROM python:3.6-slim
 
-COPY . /opt/airflow/
-
 ARG AIRFLOW_HOME=/usr/local/airflow
 ARG AIRFLOW_DEPS="all"
 ARG PYTHON_DEPS=""
-ARG buildDeps="freetds-dev libkrb5-dev libsasl2-dev libssl-dev libffi-dev libpq-dev git"
-ARG APT_DEPS="$buildDeps libsasl2-dev freetds-bin build-essential default-libmysqlclient-dev apt-utils curl rsync netcat locales"
+ARG buildDeps="freetds-dev libkrb5-dev libssl-dev libffi-dev libpq-dev git"
+ARG APT_DEPS="libsasl2-dev freetds-bin build-essential default-libmysqlclient-dev apt-utils curl rsync netcat locales"
+
+ENV PATH="$HOME/.npm-packages/bin:$PATH"
+
+RUN if [ -n "${APT_DEPS}" ]; then apt install -y $APT_DEPS; fi \
+    && apt autoremove -yqq --purge \
+    && apt clean
+
+COPY . /opt/airflow/
 
 WORKDIR /opt/airflow
 RUN set -x \
     && apt update \
-    && if [ -n "${APT_DEPS}" ]; then apt install -y $APT_DEPS; fi \
+    && if [ -n "${buildDeps}" ]; then apt install -y $buildDeps; fi \
+    && curl -sL https://deb.nodesource.com/setup_11.x | bash - \
+    && apt install -y nodejs \
```

Comment:
Can you pin the nodejs version? That way a new nodejs version won't break the image.

Reply:
I mean it is kinda pinned (to 11.x). I am not sure if we should pin this to a specific version.

Reply:
By the way the lifetime of v11 is really short [node's release schedule](https://github.com/nodejs/Release#release-schedule). Don't know if we should consider this.
Due to this I actually would pin it to the LTS (10) release instead.

Reply:
@verdan is the author of the npm stuff, therefore I've tagged him to review. Maybe he can give some guidance on the versions as well.

Reply:
I would definitely pin it to the LTS release. i.e., 10.x

Reply:
fixed